code/modules/mob/living/carbon/human/species.dm

```diff
@@ -832,11 +835,21 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				if("legs")
 					S = GLOB.legs_list[H.dna.features["legs"]]
 				if("moth_wings")
-					S = GLOB.moth_wings_list[H.dna.features["moth_wings"]]
+					if(HAS_TRAIT(H, TRAIT_MOTH_BURNT))
+						S = GLOB.moth_wings_list["Burnt Off"]
```

Comment:
is "Burnt Off" a define somewhere? if so use it

Reply:
It's not a define, none of the moth wing names are defined
![image](https://user-images.githubusercontent.com/110184118/205481885-26c78e0a-4303-4ca2-b718-d4eb940f1d04.png)